lib/include/tick/array/vector/ops_mkl.h

```diff
@@ -0,0 +1,13 @@
+
```

Comment:
Why do we implement the functions in cblas.h but not in ops_mkl.h? 
Should we have a third file that would call the functions `cblas_*` and include it in both files?
But maybe there is a reason for this dissymmetry.

Reply:
I think I have a simpler idea